src/cljs/athens/views/appearance_settings/core.cljs

```diff
@@ -0,0 +1,154 @@
+(ns athens.views.appearance-settings.core
```

Comment:
Does this need to be its own namespace?

Reply:
Up to you. Personally I prefer files much smaller than this, with more atomic concerns and less complexity.